lib/pact/provider/pact_verification_with_tags.rb

```diff
@@ -0,0 +1,13 @@
+module Pact::Provider
```

Comment:
I don't think we need a separate class for this - it's just the same as a normal verification, we've just determined the URL dynamically rather than having the user put a hardcoded one in. And, it may not have a tag.

Reply:
I can see that you'll need a consumer name for the Pact::Provider::PactVerification - just leave it null for now, and we'll work out if we need it later.